plugins/utils_operators/file_operators.py

```diff
@@ -12,6 +12,7 @@
 class DownloadFileOperator(BaseOperator):
```

Comment:
Just like in other operator classes - the changes i made here were to allow this operator to take both actual values and task info as inputs